Python/Setup/PythonToolsInstaller/Strings12.0.wxl

```diff
@@ -5,14 +5,14 @@
   <String Id="VSProductName">Visual Studio 2013</String>
   <String Id="WDProductName">Visual Studio Express 2013 for Desktop</String>
   <String Id="VWDProductName">Visual Studio Express 2013 for Web</String>
+  <String Id="ProductDescription">Python Tools for Visual Studio 2013</String>
 
-  <String Id="ProductDescription">!(loc.ProductName) !(loc.ForVisualStudio)</String>
   <String Id="DowngradeErrorMessage">A newer version of !(loc.ProductDescription) is already installed.</String>
   <String Id="NoInstallPath">!(loc.VSProductName) is required.  The free !(loc.WDProductName) and !(loc.VWDProductName) editions, as well as the required !(loc.VSProductName) Update 2, can be downloaded from http://www.visualstudio.com/.</String>
   <String Id="NoExePath">Your version of !(loc.VSProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
   <String Id="WDInstallPathButNoExe">Your version of !(loc.WDProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
   <String Id="VWDInstallPathButNoExe">Your version of !(loc.VWDProductName) is not supported.  Please install the latest update from http://www.visualstudio.com/.</String>
-  <String Id="VSPrerelease">A pre-release version of !(loc.VSProductName) was detected.  This version of Python Tools is not supported on pre-release builds of !(loc.VSProductName).  Please upgrade to !(loc.VSProductName).</String>
+  <String Id="VSPrerelease">A pre-release version of !(loc.VSProductName) was detected that is not supported by !(loc.ProductName).  Please upgrade to the final release from http://www.visualstudio.com.</String>
   <String Id="IronPythonToolsInstalled">The IronPython Tools feature of IronPython 2.7 must be uninstalled.  Python Tools replaces IronPython Tools and supports all the same features and more.</String>

```

Comment:
You updated the 14.0 product strings to use !(loc.ProductName).  May want to do the same for the 12,11,10